catalog/coo-product/catalog.yaml

```diff
@@ -15,7 +15,7 @@ name: stable
 package: cluster-observability-operator
 schema: olm.channel
 ---
-image: quay.io/redhat-user-workloads/cluster-observabilit-tenant/cluster-observability-operator-bundle@sha256:6bbed33802a729e68cd4c7022536f83354c7c5d0b39773219786b5050c76b9ca
+image: registry.redhat.io/cluster-observability-operator-bundle@sha256:6bbed33802a729e68cd4c7022536f83354c7c5d0b39773219786b5050c76b9ca
```

Comment:
I assume that I'd need to change both image/relatedImages references. Is that right, @arewm? Thanks!

Reply:
Right, when we were talking, I just mentioned the relatedImages because that is the first I saw. Any instance of this pullspec should be updated.